docs/src/docs/plugins/module-plugins.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Module Plugin System
+---
+
+An example linter can be found at [here](https://github.com/golangci/example-plugin-module-linter/settings).
+
+## The Automatic Way
+
+- define your building configuration into `.mygcl.yml`
+- run the command `golangci-lint custom` (`go run ./cmd/golangci-lint custom -v`)
```

Comment:
Don't think we need to add this to the module docs, doesn't feel like a common way to run `golangci-lint`? 

```suggestion
- run the command `golangci-lint custom`
```

Reply:
I will add the verbose version but with the binary :smile: